addon/utils/global-text-offset-to-path.ts

```diff
@@ -0,0 +1,46 @@
+import ModelNode from "@lblod/ember-rdfa-editor/model/model-node";
+import ModelTreeWalker, { FilterResult } from "@lblod/ember-rdfa-editor/model/util/model-tree-walker";
+import RawEditor from './ce/raw-editor';
+
+const VOID_NODES = ["area", "base", "br", "col", "command", "embed", "hr", "img", "input", "keygen", "link", "meta", "param", "source", "track", "wbr"];
+
+function selectTextAndVoidNodes(node: ModelNode) {
+  if (ModelNode.isModelText(node)) {
+    return FilterResult.FILTER_ACCEPT;
+  }
+  else if (ModelNode.isModelElement(node) && VOID_NODES.includes(node.type)) {
+    return FilterResult.FILTER_ACCEPT;
+  }
+  else {
+    return FilterResult.FILTER_SKIP;
+  }
+}
+
+/**
+ * This matches text offsets as found in the TextNodeWalker to a position in our virtual dom
+ * as such it assumes void nodes have a "width" of 1, being either a newline for breaks and
+ * a space for all other void nodes
+ *
+ */
+export default function globalTextOffsetToPath(editor: RawEditor, offset: number): number[] {
```

Comment:
Now that I look at this again it seems a bit weird for this to take in a whole RawEditor
a root and an offset make more sense I think? also makes it way easier to test

Reply:
hmm yes, the helper can just import ModelRange to create a range from paths.